draft-packages/illustration/KaizenDraft/Illustration/Players/LottiePlayer.spec.tsx

```diff
@@ -18,11 +18,11 @@ afterEach(cleanup)
 
 describe("<AnimatedBase />", () => {
   describe("Loading", () => {
-    beforeEach(() => {
-      mockedGetAnimationData.getAnimationData.mockResolvedValue(
-        {} as LottieAnimation
-      )
-    })
+    // beforeEach(() => {
```

Comment:
Interesting, commenting out these `beforeEach` blocks didn't make the tests fail?
If they are really unnecessary, we should delete them, rather than leaving them here as comments.

Reply:
Yeah/ Although I can see the intention of the author was to expect it to have been called? However they just put down a line of function call there.

Reply:
I have just put them back, but without `beforeEach`, and added `toHaveBeenCalled`.